src/containers/item-details/item-details.tsx

```diff
@@ -49,6 +51,11 @@ const ProductDetailsArea = ({ space = 1, className, item }: Props) => {
                 likeCount={item.likes}
                 itemId={item.itemId.slice(0, 4)}
               />
+              {isAuthor && (
+                <h6 className="bid d-flex flex-row align-items-center gap-2 my-4">
+                  You&apos;ve created a incredible NFT
```

Comment:
Change ` You&apos;ve created a incredible NFT` by  `You&apos;ve created an incredible NFT`